lib/config/scheme.js

```diff
@@ -0,0 +1,24 @@
+const configParser = require('gemini-configparser');
+
+const root = configParser.root;
+const section = configParser.section;
+const option = configParser.option;
+
+const shouldBeString = require('../metadata/validation-utils').shouldBeString;
+
+const ENV_PREFIX = `${require('../../package').name}_`;
+
+module.exports = root(
+    section({
+        username: option({
+            validate: shouldBeString
+        }),
+
+        apiToken: option({
```

Comment:
This should be just `token` (that's what it is in `api-utils.js` and the --help documentation).